apps/web/src/pages/get-started/layout/GetStartedTab.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { Grid, Timeline } from '@mantine/core';
+
+import { ProductUseCasesEnum } from '@novu/shared';
+
+import { onboardingUseCases } from '../consts';
+import { OnboardingParams } from '../types';
+import { ROUTES } from '../../../constants/routes.enum';
+import { OnboardingUseCasesTabsEnum } from '../../../constants/onboarding-tabs';
+
+interface IGetStartedTabProps {
+  usecase?: ProductUseCasesEnum;
+}
+
+export function GetStartedTab(props: IGetStartedTabProps) {
+  const navigate = useNavigate();
+  const { usecase: usecaseParam } = useParams<Record<OnboardingParams, OnboardingUseCasesTabsEnum | undefined>>();
+
+  const usecase = (props.usecase || usecaseParam?.replace('-', '_')) as ProductUseCasesEnum | undefined;
```

Comment:
not sure if we need the `OnboardingUseCasesTabsEnum` and `ProductUseCasesEnum` and do this conversion? maybe better just to use `ProductUseCasesEnum`?

Reply:
Agree here I didn't want this duplication. It happened because of the following reasons:

- We use underscores (_) in ProductUseCasesEnum to utilize our storage conversion. This enum is then reused in our DAL layer to represent different use cases on the organization entity.
- On the other hand, OnboardingUseCasesTabsEnum uses hyphens (-) to interact with the web URLs use case.

Reply:
thanks, I've checked the code before :) that's why I was suggesting to use `ProductUseCasesEnum` also in the URLs ;)

Reply:
Agree on the maintenance part etc. Do you mean to use underscores in the URL or convert without the enum?